src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ru/ResolvedSdkReference.xaml

```diff
@@ -13,8 +13,8 @@
   <StringProperty Name="Name" DisplayName="Name" ReadOnly="True" />
   <StringProperty Name="Version" DisplayName="Version" ReadOnly="True" />
   <StringProperty Name="SDKPackageItemSpec" Visible="False" ReadOnly="True" />
-  <BoolProperty Name="CopyPayload" DisplayName="Копировать полезные данные" />
-  <StringProperty Name="CopyPayloadToSubDirectory" DisplayName="Копировать полезные данные в подкаталог" ReadOnly="True" />
+  <BoolProperty Name="CopyPayload" DisplayName="Копировать полезные данные" ReadOnly="True" />
+  <StringProperty Name="CopyPayloadToSubDirectory" DisplayName="Copy Payload to Subdirectory" ReadOnly="True" />
```

Comment:
It accurately reflects the state of the source code. It is untranslated in xlf and untranslated in other languages as it was added to this file after last loc handback. Seems like it was manually translated in c462e48.